data/templates/nginx/server.tpl.conf

```diff
@@ -97,3 +97,84 @@ server {
     access_log /var/log/nginx/{{ domain }}-access.log;
     error_log /var/log/nginx/{{ domain }}-error.log;
 }
+
+# vhost dedicated to XMPP http_upload
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name xmpp-upload.{{ domain }};
+
+    location /upload {
```

Comment:
Gotta check if we're vulnerable to path-traversal issue here. We might want to add a trailing `/` I think.

Reply:
done but untested yet.